src/types/destinations/microsoft-teams.types.ts

```diff
@@ -1,6 +1,48 @@
 export type MicrosoftTeamsColor = "default" | "dark" | "light" | "accent" | "good" | "warning" | "attention";
 
+export type MicrosoftTeamsAction = {
+  type: string;
+  title: string;
+  url: string;
+};
+
+export type MicrosoftTeamsFact = {
+  title: string;
+  value: string;
+};
+
+export type MicrosoftTeamsBodyItem = {
+  type: string;
+  spacing?: string;
+  separator?: boolean;
+  facts?: MicrosoftTeamsFact[];
+  text?: string;
+  wrap?: boolean;
+  maxLines?: number;
+  fontType?: string;
+  size?: string;
+  weight?: string;
+  color?: string;
+  actions?: MicrosoftTeamsAction[];
+};
+
```

Comment:
Så du orka å type den til slutt :)

Reply:
Rider lagde typen først også forbedret jeg den enda mer 👍

Og det var nødvendig for å teste outputen